docs/GraFx-Studio/guides/template-variables-private-date/index.md

```diff
@@ -0,0 +1,41 @@
+# How to Manage Private Data
+
+You can manage [Private Data](/GraFx-Studio/concepts/private-data/) for variables in GraFx Studio. This allows you to attach custom key-value pairs to a variable, which can be used by your custom integrations, without affecting the Studio UI or end-user interface.
```

Comment:
Private data indeed doesn't affect Studio UI, but it can impact custom end-user interfaces. Integrations can use the private data to add tags to variables to trigger specific UI workflows.